service/eventer/error.go

```diff
@@ -8,10 +8,3 @@ var invalidConfigError = microerror.New("invalid config")
 func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
-
-var unexpectedStatusCode = microerror.New("unexpected status code")
-
-// IsUnexpectedStatusCode asserts unexpectedStatusCode.
-func IsUnexpectedStatusCode(err error) bool {
-	return microerror.Cause(err) == unexpectedStatusCode
-}
```

Comment:
The eventer package here is more like the eventer collection. This error is not used here.